### ISSUE TITLE
multiple swagger definition file?

### ISSUE THREAD

**Issue by yinso** [theganyo/swagger-node-runner#13]
Is there a way to load multiple swagger definition files? 

This is needed for handling API versioning as swagger defines one API version per definition file. 

Thanks.

**Comment by theganyo** [comment]
No. I'm not certain of what your looking for. Can you explain what you'd hope it would do?

**Comment by yinso** [comment]
Swagger defines one version number per file. 

So if I have version 1, version 2, and version 3 of the API - it appears that I need to publish 3 files.

So in the api/swagger/ folder, I am looking to have 

- api/swagger/
  - 1.0.yaml
  - 2.0.yaml 
  - 3.0.yaml 

And I am thinking if node runner can load all the files, or perhaps via a config being passed in. 

What do you think?

**Comment by theganyo** [comment]
I wonder if the best way to handle this would be through a front-end proxy that can route to the variations of your API based on path - or whatever criteria you choose?

**Comment by GiladShoham** [comment]
@yinso actually i did it with 3 different files and load them all successfully. 
i can help you with this if you want

**Comment by vgoloviznin** [comment]
@GiladShoham can you share the code regarding this? :)

**Comment by kazaff** [comment]
@GiladShoham how did you do?
